subiquitycore/models/network.py

```diff
@@ -196,10 +195,15 @@ def __init__(self, model, name, typ):
         self._name = name
         self.type = typ
         self.config = {}
+
+        # import done here to break a chain where anybody importing
```

Comment:
Oops! Thanks for fixing my mistake.

BTW, I learned about this syntax, not sure it would be suitable here?

```python
if typing.TYPE_CHECKING:
    import expensive_mod
```

Reply:
It was a bit of a surprise that this change broke the build.  Having the snap build logs in CI should help - the github action build log for this failure is able to show a very short log but point right to the backtrace.

We could do the TYPE_CHECKING check, I'll let you play with it after I land the snap build fix.  I guess if we're not TYPE_CHECKING we go back to `self.info = None`.